js-miniapp-sdk/CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 ## CHANGELOG
 
+### In Progress (2020-XX-XX)
+- **Feature:** Added `rakuten.miniapp.device.LOCATION` custom permission.
```

Comment:
`rakuten.miniapp.device.LOCATION` is more like an implementation detail, so I think it should be like this instead:
```suggestion
- **Feature:** Added `CustomPermissionName.LOCATION` custom permission.
```